classification.py

```diff
@@ -30,21 +30,27 @@
 
 # %%
 # Feature Selection
-selected_genes, sick_X, healthy_X = dimReducer.getNormalizedFeatures(sick,healthy,"exclude", 3)
+selected_genes, sick_X, healthy_X = dimReducer.getNormalizedFeatures(sick,healthy,"substract", 7)
+
+
+selected_genes = np.array([ 1178, 3349, 15737, 590, 10600, 232, 21125])
```

Comment:
where do these come from?

Reply:
This was the result from a long-running EA test. I just wanted to store these genes so that we can have a look on the results without waiting for the EA to run for half an hour again :)